docs/src/physics/turbulence_closures.md

```diff
@@ -25,9 +25,9 @@ Each tracer may have a unique diffusivity ``\kappa``.
 
 ## Constant anisotropic diffusivity
 
-In Oceananigans.jl, a constant anisotropic diffusivity implies a constant tensor
+A constant anisotropic diffusivity implies a constant tensor
```

Comment:
Hmm, this whole section seems to be outdated now that we have `HorizontalScalarDiffusivity` and `VerticalScalarDiffusivity`... ?